main.tf

```diff
@@ -93,7 +102,14 @@ module "nlb" {
       protocol           = "TCP"
       target_group_index = 3
     },
-  ]
+    ], var.enable_prometheus_metrics ? [
```

Comment:
I just realized that we usally write var names that enable features like so: `prometheus_metrics_enabled`. Can you adjust accordingly?

Reply:
done